index.js

```diff
@@ -18,9 +18,13 @@ console.log('hello world')
  *
  * Example Output: n/a
  */
-
 // TODO: ADD CODE HERE
 
+//Solution for challenge #1
+const chall_1_step_1 = 25;
```

Comment:
It's convention, though not required, to use "camelCase" for JS variables and functions:

`chall_1_step_1` -> `chall1Step1`